src/hotspot/share/classfile/compactHashtable.hpp

```diff
@@ -294,6 +294,7 @@ class CompactHashtable : public SimpleCompactHashtable {
     return nullptr;
   }
 
+  // Iterate through the values in the table, stopping when do_value() return false.
```

Comment:
```suggestion
  // Iterate through the values in the table, stopping when do_value() return false.
```
  // Iterate through the values in the table, stopping when iter->do_value() returns false.